### ISSUE TITLE
[Bug] Layout Children method does not called in custom layout

### ISSUE THREAD

**Issue by Sasikumar3595** [dotnet/maui#2036]
### Description

I was create the custom layout inherit from Layout<View>, add the children of the custom layout on OnMeasure method. If we add the children of the layout on OnMeasure method does not trigger LayoutChildren method, else we add the children on constructor that triggers LayoutChildren method

### Steps to Reproduce

1. Create the custom layout inherit from Layout<View>
2. Add children on OnMeasure method
3. LayoutChildren does not called so custom layout shows empty space

### Expected Behavior

- LayoutChildren will trigger

### Actual Behavior

- LayoutChildren does not triggered

**Comment by AlexeyZhukoff** [comment]
At DevExpress, we also port our controls to MAUI, and we ran into the same problem. A fix will be highly appreciated.

**Comment by hartez** [comment]
Please provide a repro project demonstrating the issue.

**Comment by Sasikumar3595** [comment]
@hartez ,

Please find the attached sample for the issue
OnMeasure and ArrangeChildren will not triggered from parent to child while InvalidateMeasure from child when the child inside on scroll view.

Sample: [MauiIssue.2.zip](https://github.com/dotnet/maui/files/7518118/MauiIssue.2.zip)

**Comment by dev-thinks** [comment]
Yes, looks like these methods are getting called in windows for me but not in Android emulator. A fix for this issue is highly appreciated.

**Comment by Sasikumar3595** [comment]
@hartez ,

Any update on this?

**Comment by hartez** [comment]
It's on the list of things to look at. We will update the issue as soon as we know something.

**Comment by VincentBu** [comment]
Hi @Sasikumar3595 , is it the expected behavior? I use the project you uploaded in #2802 and run it in Android emulator(api 28).
![Issue2036](https://user-images.githubusercontent.com/44959937/158529769-d200f161-41f9-4146-84dc-c9ccc5419083.gif)

**Comment by AswinPG** [comment]
Has there been any updates or workaround for this issue ?

**Comment by XamlTest** [comment]
Verified this issue on Visual Studio Enterprise 17.6.0 Preview 2.0. This issue does not repro on Windows 11, but reproduce on Android emulator 13 with below Project:
[2036.zip](https://github.com/dotnet/maui/files/11177144/2036.zip)
Repro video(Windows):
![Windows](https://user-images.githubusercontent.com/97647558/230567677-e8400818-dcc7-46ec-a311-970861e36e2b.gif)

Repro video(Android):
![Android](https://user-images.githubusercontent.com/97647558/230567695-949223fb-3b4f-4e83-a635-6805307cb711.gif)

**Comment by vishnumenon2684** [comment]
The issue seems to be not reproducing with the latest Maui NuGet version based on net10  (10.0.0-rc.2.25504.7) on Android platform. Tested on emulator based on pixel 8 pro running api 36.

https://github.com/user-attachments/assets/d532e9f8-08e7-42db-b96b-dd8785824f27
